packages/DataTable/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@paprika/data-table",
+  "version": "0.1.0",
+  "description": "A virtualized table component.",
+  "author": "@paprika",
+  "license": "MIT",
+  "main": "lib/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/acl-services/paprika.git",
+    "directory": "packages/DataTable"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "exports": {
```

Comment:
why do we need a separate exports? I thought that `"main": "lib/index.js",` would be enough

Reply:
I was going to try out if developers can `import RowHeightHelper from '@paprika/data-table/RowHeightHelper'`
but maybe not better than `import {RowHeightHelper} from '@paprika/data-table'`
will delete it 😅